deployments/deploy-tests.ps1

```diff
@@ -0,0 +1,145 @@
+try{
+    Import-Module .\azure\az-cloud-context\az-cloud-context.psd1
+    Import-Module .\azure\resources\key-vault\key-vault.psd1
+    Import-Module .\azure\resources\log-analytics\log-analytics.psd1
+    Import-Module .\powerform\pf-deployment-context\pf-deployment-context.psd1
+
+    Get-Command -Module log-analytics
+    Get-Command -Module key-vault
+    Get-Command -Module pf-deployment-context
+    Get-Command -Module az-cloud-context
+
+    ##############################################################################################################
+    #
+    #               Deployment Context can drive naming defaults 
+    #
+    #############################################################################################################
+
+    #Still need to build this. The thought behind this is that a user wants to deploy resources out with best practices/enterprise standards
+    #baked into the module.  By defining the deploy context parameters, the names will be created, therefore it knows
+    #the RG, Sub, Resources names.  All options are going to be the defaults, so just build the definitions and deploy
+
+    #$deployContext = New-PfDeploymentContext -CompanyAbbreviation "MYAEA" -GroupAbbreviation "KKZH" -TeamAbbreviation "ADM" -EnvironmentLetter "D"
```

Comment:
this is really cool

Reply:
Yeah, I think the biggest thing is finding and naming the right parameters off of the PFDeploymentContext.  Then those can drive everything for that current context.  So we really need no options for any resources, subs, or rgs if we trust powerform's default enterprise standards enough.  We just say give me a KV, a LA, and a SQL DB.  And it just puts everything in there for you.